src/test/kotlin/org/gradle/script/lang/kotlin/accessors/ProjectSchemaAccessorsIntegrationTest.kt

```diff
@@ -10,10 +10,46 @@ import org.junit.Assert.assertFalse
 import org.junit.Test
 
 import java.io.File
+import java.nio.charset.StandardCharsets.UTF_8
 
 class ProjectSchemaAccessorsIntegrationTest : AbstractIntegrationTest() {
 
     @Test
+    fun `can configure deferred configurable extension`() {
+        withBuildScript("""
```

Comment:
I'd add a blank line between the back-quoted function name and `withBuildScript(...)`.